buildkite/src/Command/RunWithPostgres.dhall

```diff
@@ -46,12 +45,25 @@ let runInDockerWithPostgresConn
               : Text
               = "\\\$BUILDKITE_BUILD_CHECKOUT_PATH"
 
+          -- read the latest archive data version from the version file 
```

Comment:
My intention of this Command (RunWithPostgres.dhall) was that it is loosely coupled with dependency to mina internals. That's why it accept docker and init script  as text. After your changes it become more tightly coupled with o1labs ggloud. I would love to see some layer above this file which deals with all mina related stuff. In this case it should download locally archive test data and run RunWithPostgres.dhall command.